resources/assets/uad_lists.json

```
@@ -37622,5 +37622,77 @@
     "neededBy": [],
     "labels": [],
     "removal": "Advanced"
+  },
+  "com.nim.rogers":{
+    "list": "Carrier",
+    "description": "Texture, a digital magazine service created by Rogers Media. Discontinued in 2019.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.Rogers.MyRogersTab":{
+    "list": "Carrier",
+    "description": "Appears to be the tablet version of MyRogers (https://play.google.com/store/apps/details?id=com.fivemobile.myaccount), an app to manage your account with Canadian carrier Rogers.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.samsung.android.app.colorblind":{
+    "list": "Oem",
+    "description": "This app provides colorblind accessibility features for Samsung devices, helping users with color vision deficiencies by adjusting display colors and improving visibility.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```

This could be very useful for colorblind people and it's not outright negative, see https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/wiki/FAQ. Do you agree?

---

> This could be very useful for colorblind people and it's not outright negative, see https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/wiki/FAQ. Do you agree?

Personally I found the feature didn't work all that well, and it was kinda hidden in the settings, but I could see why it may be useful to some colourblind people, I'll change it to Advanced shortly.

---

Thanks.

---

Changed.